### PR TITLE
[nms] Update @fbcnms/alarms to 0.2.3

### PR DESCRIPTION
**Pull request by andreilee**
Signed-off-by: Andrei Lee <andreilee@fb.com>

## Summary

Updating the `@fbcnms/alarms` package from version `0.1.11` to `0.2.3`. This update is a transition towards components that are shared between Magma, and Terragraph NMS.

As part of this update, components `Alarms.js` and `AlertRules.js` are deleted, which were originally copied from `@fbcnms/alarms` and modified in Magma codebase. As an attempt to move towards shared code again, the shared component is used as intended.

One existing issue with this update is that when no alert rules are configured, a button is shown for adding an alert rule, but the route is hardcoded, and does not work for Magma. This will be updated in `@fbcnms/alarms` for a later version.

**Updated Alarms Page**
Screen Shot 2021-05-12 at 4 26 25 PM
Screen Shot 2021-05-12 at 4 26 05 PM
Screen Shot 2021-05-12 at 4 26 01 PM
Screen Shot 2021-05-12 at 4 25 57 PM

**Old Alarms Page**
Screen Shot 2021-05-12 at 4 28 02 PM
Screen Shot 2021-05-12 at 4 28 55 PM

## Test Plan

- Synced predefined alerts
- Viewed alarms page, alert rules page, and receivers page
- Created and viewed rules, edited rules

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by karthiksubraveti — COMMENTED** [review]
I'm unable to find any differences here? What do we see with new component ? Is there any stylistic difference ?

**Comment by karthiksubraveti** [comment]
> I'm unable to find any differences here? What do we see with new component ? Is there any stylistic difference ?

ignore this comment

File: orc8r/cloud/docker/metrics-configs/alertmanager.yml [hunk 1]
```diff
@@ -11,9 +11,14 @@ route:
   receiver: null_receiver
   group_by:
   - alertname
+  routes:
```
Comment (karthiksubraveti) [inline orc8r/cloud/docker/metrics-configs/alertmanager.yml]: Why this change ?

File: nms/app/packages/magmalte/app/views/alarms/AlarmsDashboard.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    );
+  };
+  return (
+    <>
+      <TopBar header={'Alarms'} tabs={tabs} />
+      <div className={classes.root}>
+        {isSuperUser && <SyncAlertsButton />}
+        <Alarms
+          makeTabLink={makeTabLink}
+          disabledTabs={['alerts', 'rules', 'suppressions', 'routes', 'teams']}
```
Comment (karthiksubraveti) [inline nms/app/packages/magmalte/app/views/alarms/AlarmsDashboard.js:103]: What does this disabled tabs mean ? If we disable alerts/rules/teams and still use it, would that work ? Also is there any behavior change for adding alert receivers ?

**Review by karthiksubraveti — COMMENTED** [review]
Once this merges, can you also update documentation with latest screenshots to the keep the docs current. Thanks

File: nms/app/packages/magmalte/app/views/alarms/AlarmsDashboard.js (hunk 2 above)
Comment (andreilee) [reply nms/app/packages/magmalte/app/views/alarms/AlarmsDashboard.js:103]: The styling doesn't make all the tabs visible at once, so I hide the tabs. This has no behavior change

File: orc8r/cloud/docker/metrics-configs/alertmanager.yml (hunk 1 above)
Comment (andreilee) [reply orc8r/cloud/docker/metrics-configs/alertmanager.yml]: Oh I should remove this

**Review by karthiksubraveti — APPROVED** [review]
Changes look good. Have you tested the channel receivers ? Can we add a slack receiver, can we check to see if we have any regression here. Thanks.

**Review by karthiksubraveti — APPROVED** [review]
